core/src/main/java/hudson/model/RunMap.java

```diff
@@ -229,8 +229,11 @@ protected R retrieve(File d) throws IOException {
                 return b;
             } catch (IOException e) {
                 LOGGER.log(Level.WARNING, "could not load " + d, e);
+                throw new IOException();
```

Comment:
In that case delete the log line above, since `AbstractLazyLoadRunMap.load` already logs the same information.